src/forms/reset-password-popup/reset-password/index.jsx

```diff
@@ -77,7 +77,9 @@ const ResetPasswordPage = () => {
   useEffect(() => {
     if (formErrors && Object.keys(formErrors).length > 0 && errorRef.current) {
       setTimeout(() => {
-        errorRef.current.focus();
+        if (errorRef.current) {
```

Comment:
This condition is already handle above, `errorRef.current`, Also we can remove time out.

Reply:
done